StaticAnalyzer/views/android/manifest_analysis.py

```diff
@@ -1269,7 +1269,8 @@ def read_manifest(app_dir, tools_dir, typ, binary):
 
             args = [settings.JAVA_PATH + 'java', '-jar', cp_path, manifest]
             dat = subprocess.check_output(args)
-            dat = dat.replace(b"\n", b"")
+            #dat = dat.replace(b"\n", b"")
+            dat = dat.decode()

```

Comment:
What about? 
`dat.decode("utf-8", "ignore") `

Reply:
if you re-add dat.replace(b"\n", b"")
it will produce
![image](https://user-images.githubusercontent.com/27353689/36092474-c7a2f186-0fe7-11e8-998c-71ef368629c4.png)

Reply:
I understand, I am asking about line 1273. Shall we use `dat.decode("utf-8", "ignore")` ? Because that looks more failsafe. what do you think?

Reply:
agree with you to  use dat.decode("utf-8", "ignore")